modules/ztp-creating-siteconfig-custom-resources.adoc

```diff
@@ -0,0 +1,351 @@
+a| * BMC Secret authenticates into the target bare metal machine using its username and password.
+* Image Pull Secret contains authentication information for the {product-title} image installed on the target bare metal machine.
+
+|ClusterImageSet
+|Contains {product-title} image information such as the repository and image name.
+|Passed into resources to provide {product-title} images.
+|===
+
+.Prerequisites
+
+* Enable Assisted Installer Service.
+
+* Ensure network connectivity:
+** The container within the hub must be able to reach the Baseboard Management Controller (BMC) address of the target bare metal machine.
+
+** The managed cluster must be able to resolve and reach the hub’s API `hostname` and `{asterisk}.app` host name.
+Example of the hub’s API and `{asterisk}.app` host name:
++
+[source,terminal]
+----
+console-openshift-console.apps.hub-cluster.internal.domain.com
+Api.hub-cluster.internal.domain.com
+----
+
+** The hub must be able to resolve and reach the API and `{asterisk}.app` host name of the managed cluster.
+Here is an example of the managed cluster’s API and `{asterisk}.app` host name:
++
+[source,terminal]
+----
+console-openshift-console.apps.sno-managed-cluster-1.internal.domain.com
+Api.sno-managed-cluster-1.internal.domain.com
+----
+
+** A DNS Server that is IP reachable from the target bare metal machine.
+
+* A target bare metal machine for the single node cluster with the following hardware minimums:
+
+** 4 CPU or 8 vCPU
+** 32 GiB RAM
+** 120 GiB Disk for root filesystem
+
+* Create a ClusterImageSet for each specific cluster version that needs to be deployed. A ClusterImageSet has the following format:
++
+[source,yaml]
+----
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  name: openshift-4.8.0-rc.0 <1>
+spec:
+   releaseImage: quay.io/openshift-release-dev/ocp-release:4.8.0-x86_64 <2>
+----
+<1> `name` is the descriptive version that you want to deploy.
+<2> `releaseImage` needs to point to the specific release image to deploy.
+
+* When working in a disconnected environment, the release image needs to be mirrored. Use this command to mirror the release image:
++
+[source,terminal]
+----
+oc adm release mirror -a <pull_secret.json>
+--from=quay.io/openshift-release-dev/ocp-release:{{ mirror_version_spoke_release }}
+--to={{ provisioner_cluster_registry }}/ocp4 --to-release-image={{
+provisioner_cluster_registry }}/ocp4:{{ mirror_version_spoke_release }}
+----
+
+* You need to mirror the ISO and `rootfs` used to generate the spoke cluster ISO. Store the images in an http server and configure the settings to pull images from there.
++
+The images must match the version of the ClusterImageSet. To deploy a 4.8.0 version, the `rootfs` and ISO need to be 4.8.0 as well.
+
+
+.Procedure
+
+Use this procedure to create and provision one single node cluster. To provision additional clusters, repeat this procedure for each cluster.
+
+. Create the namespace for the single node managed cluster:
```

Comment:
```suggestion
. Create the namespace definition for the single-node managed cluster:
```

Reply:
You're not actually creating the namespace in this step. Please take another look.